readthedocs/templates/sphinx/conf.py.conf

```diff
@@ -10,7 +14,7 @@ source_parsers = {
         }
 master_doc = 'index'
 project = u'{{ project.name }}'
-copyright = u'2016'
+copyright = str(datetime.now().year)
```

Comment:
Although, should be better to generate this on _template parse time_  (using {{ time }}) rather than build time?